openlibrary/components/MergeUI/MergeRow.vue

```diff
@@ -43,12 +65,27 @@ export default {
             type: Array,
             required: true
         },
+        editions: {
+            type: Array,
+        },
+        lists: {
+            type: Object,
+        },
+        bookshelves: {
+            type: Object,
+        },
+        ratings: {
+            type: Object,
+        },
```

Comment:
It would nice if this merge ui was a bit more agnostic about what exactly its merging, but this makes more sense with the current state of the world